src/main/java/org/testng/ITestNGMethod.java

```diff
@@ -211,4 +211,8 @@
   default boolean isDataDriven() {
     return false;
   }
+
+  default Object getRawInstance() {

```

Comment:
Maybe some javadoc is missing. What's the difference between raw instance and instance?

Reply:
Renamed this for better readability